data/categories.txt

```diff
@@ -4721,43 +4721,6 @@ zh-Hans:赌博
 zh-Hant:賭博
 
 amenity-gambling|@gambling
-en:Gambling
```

Comment:
Did you test if it works like this?

Reply:
yeap, it works
and there are other "empty" entries like this, e.g. `shop|@category_shop`